forte_wrapper/elastic/elastic_search_processor.py

```diff
@@ -43,12 +43,32 @@ def initialize(self, resources: Resources, configs: Config):
 
     @classmethod
     def default_configs(cls) -> Dict[str, Any]:
+        """
+        This defines a basic config structure for ElasticSearchProcessor.
+        :return: A dictionary with the default config for this processor.
```

Comment:
Sorry I just notice the format for `return` is wrong. You probably copied it from AllenNLP, but that's wrong too.

Here is the correct example (following Google style return value)
https://github.com/asyml/forte/blob/master/forte/data/data_pack.py#L171

Here are more examples:

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
do you mean this style https://github.com/asyml/forte-wrappers/blob/main/forte_wrapper/vader/sentiment_analysis.py#L64  or this style https://github.com/asyml/forte-wrappers/blob/main/forte_wrapper/elastic/elastic_indexer.py#L159 ? I found both in the example.

Reply:
These two are good, but the current docstring contains this `:returns:` , which is another style that we didn't use.

Reply:
Here is the description of our style: https://google.github.io/styleguide/pyguide.html#doc-function-returns